wire-protoc-compatibility-tests/src/test/java/com/squareup/wire/Proto3WireProtocCompatibilityTests.kt

```diff
@@ -746,10 +788,7 @@ class Proto3WireProtocCompatibilityTests {
         |  "packSint64": ["-9223372036854775808", "-9223372036854775808"],
         |  "packFixed64": ["9223372036854775808", "9223372036854775808"],
         |  "packSfixed64": ["-9223372036854775808", "-9223372036854775808"],
-        |  "oneofInt64": "-9223372036854775808",
-        |  "mapInt64Sfixed64": {
```

Comment:
what happened here

Reply:
Removed them because we don't support map yet.